apps/orams/index.js

```diff
@@ -3,9 +3,9 @@ import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
 import { BrowserRouter } from 'react-router-dom'
 
-import HomepageHeader from './components/HomepageHeader/HomepageHeader'
-import Banner from '../shared/Banner/Banner'
-import Footer from '../shared/Footer/PageFooter'
+import Banner from 'shared/Banner/Banner'
```

Comment:
we should still follow import format from style guide:
- absolute imports
- blank line
- relative imports
- blank line
- css modules